bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/ThingFactory.java

```diff
@@ -50,7 +50,9 @@
  */
 public class ThingFactory {
 
-    private static Logger logger = LoggerFactory.getLogger(ThingFactory.class);
+    private static Logger logger() {
+        return LoggerFactory.getLogger(ThingFactory.class);
```

Comment:
Hm, this creates a logger instance on every single call - I think this can potentially cause memory and performance issues. I would therefore really prefer to keep the static version (at least as long as we really do not see any problem that is potentially caused by it).